'ui_password' generator can sometimes add a backslash (\) causing invalid string escape

### Expected Behavior
Generate a password with no errors or crashing

### Actual Behavior
On first startup I got this as my password:
`\AsianStep74`
or
`ui_password = "admin:\AsianStep74"`
now at first glance it looks fine except for the weird backslash

well...

`[ERROR] 2023/08/21 21:20:54 Using Config File: /config/notifiarr.conf: getting config: config file: unmarshaling file /config/notifiarr.conf: toml: line 17 (last key "ui_password"): invalid escape in string '\A'`

### Steps to reproduce

- Reinstall Notifiarr until  it generates you a password
- or just add a backslash to your password...

### Fix
I see two fixes for this

1. If you have control over the generator then disable backslashes...

if you don't have control then the next two options.

2. Search for the backslashes and add a second one `\ -> \\`
this is what I did because I was curious if i would get locked out of the web ui, I did not.
I set the password as `\\AsianStep74` which fixed that crashing and enabled me to login via `\AsianStep74`

3. Search for all backslashes and eliminate them...

PS. Sorry i was bored when writing this so I went all out, and yes that password was actually generated.

--- a/pkg/configfile/password.go
+++ b/pkg/configfile/password.go
@@ -144,7 +144,7 @@ func GeneratePassword() string {
 	}
 
 	pieces[1] = strconv.Itoa(rand.Intn(89) + 10)
-	punctuation := strings.Split(`!@#$%^&*+=/<>\|~`, "")
+	punctuation := strings.Split(`!@#$%^&*+=/<>|~`, "")
 	pieces[2] = punctuation[rand.Intn(len(punctuation))]
 
 	pieces[3] = words[rand.Intn(len(words))]

